common/app/common/TrailsToRss.scala

```diff
@@ -131,9 +131,10 @@ object TrailsToRss extends implicits.Collections {
         .filterNot(_.config.excludeFromRss)
         .flatMap(_.all)
         .filter{
-        case _: LinkSnap => false
-        case _ => true}
-        .distinctBy(_.id)
+          case _: LinkSnap => false
+          case _ => true}
+        .filter(_.maybeContentId.isDefined)
```

Comment:
This makes no sense... Oh you mean in relation to your first comment.
No, I don't like `collect` in this case, as there is no `collectNot`.